shared/iothub_csharp_shared.sln

```diff
@@ -107,14 +81,14 @@ Global
 		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Debug|x64.Build.0 = Debug|Any CPU
 		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Debug|x86.Build.0 = Debug|Any CPU
-		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|Any CPU.ActiveCfg = Release_Delay_Sign|Any CPU
-		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|Any CPU.Build.0 = Release_Delay_Sign|Any CPU
-		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|ARM.ActiveCfg = Release_Delay_Sign|Any CPU
-		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|ARM.Build.0 = Release_Delay_Sign|Any CPU
-		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|x64.ActiveCfg = Release_Delay_Sign|Any CPU
-		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|x64.Build.0 = Release_Delay_Sign|Any CPU
-		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|x86.ActiveCfg = Release_Delay_Sign|Any CPU
-		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|x86.Build.0 = Release_Delay_Sign|Any CPU
+		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|Any CPU.ActiveCfg = Release|Any CPU
+		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|Any CPU.Build.0 = Release|Any CPU
+		{91DFB837-D8A3-4F54-AE0D-45C95ACD0C2A}.Release_Delay_Sign|ARM.ActiveCfg = Release|Any CPU
```

Comment:
>Release [](start = 76, length = 7)

not sure if this is intended?  and why?

Reply:
Most likely introduced by the update to the new UWP.